cocos/scripting/lua-bindings/auto/lua_cocos2dx_auto.cpp

```diff
@@ -36582,7 +36582,7 @@ int lua_cocos2dx_EventMouse_getMouseButton(lua_State* tolua_S)
             tolua_error(tolua_S,"invalid arguments in function 'lua_cocos2dx_EventMouse_getMouseButton'", nullptr);
             return 0;
         }
-        int ret = cobj->getMouseButton();
```

Comment:
please don't modify auto generated files, the file will be auto generated in CI machine